config/p2p.go

```diff
@@ -17,17 +17,25 @@ type P2PConfig struct {
 	GossipedBlocksCacheSize int `mapstructure:"p2p_gossiped_blocks_cache_size"`
 	// Time interval a node tries to bootstrap again, in case no nodes connected
 	BootstrapRetryTime time.Duration `mapstructure:"p2p_bootstrap_retry_time"`
+	// Param used to enable block sync from p2p
+	BlockSyncEnabled bool `mapstructure:"p2p_blocksync_enabled"`
+	// Time interval used by a node to request missing blocks (gap between cached blocks and local height) on demand from other peers using block-sync
+	BlockSyncRetrieveRetryTime time.Duration `mapstructure:"p2p_blocksync_retrieve_retry_time"`
```

Comment:
imo `retry` signify something that failed (which is not necessarily the case) and not an ongoing routine process.
 I'd name it `p2p_blocksync_block_request_interval` or something like that.